SwiftPMTests/analytics-import-test/analytics-import.swift

```diff
@@ -21,7 +21,7 @@ import FirebaseAnalytics
 
```

Comment:
Consider adding an import to the new library in this test and any other possible tests.

Reply:
The new library test is tricky. The API is exported in FirebaseAnalytics. Without the OnDeviceConversion library, the new API will still be visible, but just an no-op. To verify the internal behavior is also difficult here. Will discuss internally to see how to create tests here.